docs/help/account/billing.md

```diff
@@ -0,0 +1,44 @@
+# Billing
+
+The Billing section of your account provides access to the following items:
+
+- information about your current plan
+- your invoice history
+- your billing details
+
+Bump.sh offers monthly or annual billing. Payment is by credit card only; we do not offer any alternatives at this time. We use Stripe to handle all payments, so we never store your bank details.
+
+## Invoices
+
+### Edit Your Billing Information
+
+You can edit the following billing information so that it appears on your upcoming invoices:
+
+- Company name
+- Address
+- VAT Number
+
+:::info
+Edited information will only appear on future invoices; it is not possible to edit the details of an already created invoice.
```

Comment:
Not my best English here but you get the idea.

```suggestion
Updating your billing information will only apply to future invoices; it is not possible to edit the details of an already created invoice.
```

Reply:
Modifying but not 100% sure.

Reply:
LGTM 👍